zerver/lib/actions.py

```diff
@@ -2020,18 +2020,29 @@ def set_default_streams(realm, stream_names):
                'domain': realm.domain,
                'streams': stream_names})
 
+
+def notify_default_streams(realm):
+    # type: (Realm) -> None
+    event = dict(
+                type = 'update_default_streams',
```

Comment:
(Also, the whitespace still looks inconsistent with how we create event dicts elsewhere in the file)